src/pages/workspace/categories/WorkspaceCategoriesPage.tsx

```diff
@@ -253,6 +258,8 @@ function WorkspaceCategoriesPage({policy, policyCategories, route}: WorkspaceCat
 
     const isLoading = !isOffline && policyCategories === undefined;
 
+    const shouldShowEmptyState = !categoryList.some((category) => category.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE) && !isLoading;
```

Comment:
When delete category offline, it should be crossed out instead of disappearing.
https://github.com/Expensify/App/issues/43127